mobify/astro-scaffold
# Add Launch Screen

Add a launch image to the scaffold.

JIRA: https://mobify.atlassian.net/browse/HYB-945
Linked PRs: https://github.com/mobify/astro/pull/671

## Changes
- On Android. Add a theme to the `MainActivity`
- Dismiss the launch image after the main view plugin is set

## How to test-drive this PR
- Launch the app on Android. You will see a launch image with the android app icon.
- Launch screen goes away when the main view is shown

## TODOS:
